utils/countryUtils.js

```diff
@@ -1,39 +1,32 @@
 const stringUtils = require('./stringUtils');
 const countryData = require('./countries');
 
-/**
- * Transform null values to 0 (DEEPLY TRANSFORM)
- * @param {Object} object country/continent object
- * @returns  {Object} 		transformed object
- */
-const transformNull = (object) => {
-	if (typeof object !== 'object') {
-		return object;
+
+const convertNullToZero = (continentOrCountry = null) => {
```

Comment:
good naming

Reply:
Or ``nullToZero` as I see most developers do, but I prefer functions start with a verb to refer the action.